### DISCUSSION TITLE
CoPilot suggestion containing name of Github user

### DISCUSSION THREAD

**Discussion by iEns** [community/community#7948] [Copilot Conversations]
Got this suggestion today after typing // in a C program:

`// Copyright (c) 2014, 2015, 2016, 2017, 2018, 2019, 2020 Adrian Kennard`

A quick search confirms that Mr. Kennard is a Github contributer
The FAQ suggests there is a rudimentary filter for personal identifiable information in the training dataset... Maybe copyright notices should be included in this filter?

**Comment by MulleDK19** [comment]
Well, the name is already public, by choice, by the person themselves, so it's not like it's really an issue.
